### PR TITLE
Sprint 158 Usman

### PR DESCRIPTION
**Pull request by UsmanLhrPk**
- [x] AVIARY-4836 - fix: MOCA unable to update some OHMS Indexes

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
   end
   devise_for :admins
   root 'home#index'
-  get '/collection', to: 'catalog#index', as: :org_collection
+  get '/collection', to: 'catalog#index', as: :org_collection, constraints: RestrictJsonRequest.new
```
Comment (fayzan-cbr) [inline config/routes.rb:23]: Any specific reason why we are applying a constraint on this route?

Comment (UsmanLhrPk) [reply config/routes.rb:23]: Because this is only allowed with Content-Type `JSON`. It will throw an error with other content-types

Comment (fayzan-cbr) [reply config/routes.rb:23]: Okay

Comment (fayzan-cbr) [reply config/routes.rb:23]: Do we also need to check for format? `request.format == :json` or is that not required?

**Review by fayzan-cbr — APPROVED** [review]

**Pull request merged**